pkgs/tools/virtualization/awless/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
 
 buildGoPackage rec {
   pname = "awless";
```

Comment:
> Bluntly said unfortunately, this project has stopped, no development is done on it and it is not worth submitting PRs. I was a contributor but I am no longer in the company (Wallix)

2021-06-02

https://github.com/wallix/awless/issues/297#issuecomment-853186227

last commit was 2018-12-10!

---

I think we should probably remove awless

The maintainers of this derivation haven't committed to nixpkgs since 2017 and 2019 respectively although @swdunlop has commented recently so :wave:
Any thoughts?

Reply:
Awless still works, it just doesn't understand any AWS features newer than 2018.  I do not know why it stopped, outside of the comments we see in wallix/awless#297.  I will dig in, see if I can root out why x/sys/unix is not building on linux-aarch64 with Go 1.17.10 and how wide the damage is.

Reply:
The dependencies are super old and who knows what vulnerabilities are in there since most scanners dont consume dep format

Reply:
> The dependencies are super old and who knows what vulnerabilities are in there since most scanners dont consume dep format
> 
> There are packages that have had commits in the last 2 years we've dropped as unmaintained, this one is almost 4
> 
> If someone wants to migrate it to go modules and update the dependencies then sure

Sorry, I thought this was related to ZHF.  If this was really about airing frustration about awless not being updated for years, I agree.  But I do not have the resources or time for a hard fork of someone else's project.

Reply:
ZHF means the packages that are broken come into view. When looking at the derivation and the upstream project its just better to drop it as unmaintained rather than mark it broken

#172281

If you as the maintainer want to keep it then fair enough. I dont think there's an explicit nixpkgs policy around dead projects

Reply:
Yeah, either mark it broken and remove in 1-2 releases if nothing happens, or just drop immediately.